### PR TITLE
Zproject regen 201905

### PR DESCRIPTION
**Pull request by aquette**
Signed-off-by: Arnaud Quette <ArnaudQuette@Eaton.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,16 +11,12 @@ cache:
 os:
 - linux
 
-# Note: some packages or dependencies may require extended permissions
```
Comment (aquette) [inline .travis.yml]: note to @jimklimov and @perrettecl This is the only block I've a doubt on (sudo + dist). Interested in your feedback on this, thx

Comment (perrettecl) [reply .travis.yml]: On my side, I kept the new generated version of this section (without sudo : false and with trusty)

Comment (aquette) [reply .travis.yml]: indeed what I saw and mapped to.
however, the one from @jimklimov on fty-example (though dating from 27 days) was still referring to it, hence /me puzzled and possibly missing something ;)

Comment (jimklimov) [reply .travis.yml]: I think the setting dealt with Travis defaults at the time of original recipe generation, to make the build results more deterministic (there were differences for builds in docker or not, and with sudo or not, trading off some manageability over package sets and root actions IIRC vs. deploy speeds).

In practice it is now expected to matter for our `*-rest` components that have to fix up SASL pkg-config files as root, and not likely matters for others.

**Pull request merged**